RecoBTag/DeepFlavour/python/DeepFlavour_cff.py

```diff
@@ -0,0 +1,10 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoBTag.DeepFlavour.DeepFlavourTagInfos_cfi import pfDeepFlavourTagInfos
+from RecoBTag.DeepFlavour.DeepFlavourJetTags_cfi import pfDeepFlavourJetTags
+from CommonTools.PileupAlgos.Puppi_cff import puppi
+from PhysicsTools.PatAlgos.slimming.primaryVertexAssociation_cfi import primaryVertexAssociation
+
+
+pfDeepFlavourTaskNew = cms.Task(puppi, primaryVertexAssociation,
```

Comment:
puppi is not a part of RECO step, it is a part of miniAOD.
 puppi often gets tuned up only in time for (re)miniAOD.
So, if deepFlavour needs to depend on puppi, it should be moved to miniAOD step, otherwise it becomes "obsolete" at creation point because the officially used puppi will be made later (these steps are separated in MC and rerecos).

Reply:
```New``` in ```pfDeepFlavourTaskNew``` does not have a permanent meaning.
The name should be disambiguated in a different way